var/spack/repos/builtin/packages/suite-sparse/package.py

```diff
@@ -96,9 +96,12 @@ def install(self, spec, prefix):
 
         # Intel TBB in SuiteSparseQR
         if 'tbb' in spec:
+            # NOTE: if the tbb_453 patch *stops* being applicable
+            # then the ld flags for tbb should be updated correspondingly
             make_args.extend([
                 'SPQR_CONFIG=-DHAVE_TBB',
                 'TBB=-L%s -ltbb' % spec['tbb'].prefix.lib,
+                'CFLAGS+=-I%s' % spec['tbb'].prefix.include
```

Comment:
We have been monkeying around with build_environment lately. I noticed that some build deps weren't showing up in my PATH. Might want to investigate whether or not things are working properly.